### ISSUE TITLE
Setting this up on my own

### ISSUE THREAD

**Issue by CptPlastic** [exodus4d/pathfinder#914]
I tried detting this up on my own im not sure what I did wrong but there seems to be missing files. 

When I set the SSO stuff up the directions say to set a path that in not there.
https://foxyellow.com/

What else am I missing?

**Comment by zap2zero** [comment]
Did you follow the instructions on the [Wiki](https://github.com/exodus4d/pathfinder/wiki/SSO-ESI)?

**Comment by CptPlastic** [comment]
Yeah I just dont know where the sso path is supposed to be or what im doing wrong. Its like when it redirects it goes to a place that I cant find. 
I see this main page https://foxyellow.com/
but when I click set up it tells me the path is not there. Im wondering if I need to do something with the routes?

**Comment by zap2zero** [comment]
You go to https://developers.eveonline.com/ and register PF as an app of yours and the callback url should be https://pathfinder.foxyellow.com/sso/callbackAuthorization as mentioned in the wiki and thats it.

**Comment by CptPlastic** [comment]
I did all that. More over though the **/setup** is not working.

**Comment by CptPlastic** [comment]
I was able to make some progress. I had to fix MOD-Rewrite is was not working correctly in AWS. I simply added it only minus all the other stuff on it that seemed to be for a local environment. Now I am digging through the settings but its working. So if you are running this in AWS you need to only have this is your .htaccess or default with njinx.

```
RewriteEngine On
RewriteCond %{HTTP_HOST} !^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$

RewriteCond %{ENV:REDIRECT_STATUS} ^$
RewriteRule ^(lib|tmp)\/|\.(ini|php)$ - [R=404]

RewriteCond %{REQUEST_FILENAME} !-l
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule .* index.php [L,QSA]
RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization},L]

<ifmodule mod_expires.c>
    
    <Filesmatch "\.(eot|woff2|woff|ttf|ttf|svg)$">
        ExpiresActive on
        ExpiresDefault "access plus 1 month"
        Header append Cache-Control "public"
    </Filesmatch>

    
    <Filesmatch "\.(jpg|jpeg|png|gif|swf|ico|svg)$">
        ExpiresActive on
        ExpiresDefault "access plus 1 year"
        Header append Cache-Control "public"
        FileETag None
        Header unset ETag
    </Filesmatch>

   
    <Filesmatch "\.(css)$">
        ExpiresActive on
        ExpiresDefault "access plus 1 month"
    </Filesmatch>

    
    <Filesmatch "\.(js|map)$">
        ExpiresActive on
        ExpiresDefault "access plus 1 year"
        Header append Cache-Control "public"
        FileETag None
        Header unset ETag
    </Filesmatch>

    
    <Filesmatch "\.(htm|html)$">
        ExpiresActive on
        ExpiresDefault "access plus 1 week"
    </Filesmatch>
</ifmodule>
```

**Comment by CalMatar** [comment]
Are you talking about the SSO path from the h5bp NGINX setup?

**Comment by CptPlastic** [comment]
For Nginx this works
```
# nginx configuration

location / {
  if (!-e $request_filename){
    rewrite ^(.*)$ /index.php break;
  }
}
```

**Comment by CptPlastic** [comment]
A note about my environment its in AWS and **I do not use** "WWW" so I removed that rewrite. I also set up SSL prior to all this and everything redirects to HTTPS. 

This stumped me because the htaccess that come with the install really need to be slimmed down to avoid confusion and possibly include basic minimal use cases as a default.

**Comment by Caffe1neAdd1ct** [comment]
I'd agree with simplifying the htaccess, our environment is also not `www.` which made setting this up interesting, slightly less so as I don't allow htaccess files, so just added the relevant settings to the virtual host as needed.

**Comment by GQuantrill** [comment]
I run from docker and end up just doing a reverse proxy on the IP of the docker container
